apps/web/src/ee/clerk/providers/ClerkProvider.tsx

```diff
@@ -0,0 +1,52 @@
+import { PropsWithChildren } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { CLERK_PUBLISHABLE_KEY, IS_CLERK_AUTH_ENABLED } from '../../../config/index';
+import { ClerkProvider as _ClerkProvider } from '@clerk/clerk-react';
+import { useColorScheme } from '@novu/design-system';
+import { dark } from '@clerk/themes';
+
+const ClerkModalElement = {
+  modalContent: {
+    width: '80rem',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Use token or add TODO for using token‏